glide.yaml

```diff
@@ -31,7 +31,7 @@ import:
 - package: github.com/olekukonko/tablewriter
 - package: github.com/pkg/browser
 - package: github.com/spf13/viper
-  repo: https://github.com/jimmidyson/viper.git
+  repo: https://github.com/spf13/viper.git
```

Comment:
If we don't use a custom repo we don't need the explicit _repo_ part. But we might need to set a version. See also https://github.com/minishift/minishift/blob/master/docs/developing.md#updating-dependencies